### ISSUE TITLE
Bad color scheme for window-numbering in spaceline when running Emacs server (daemon mode)

### ISSUE THREAD

**Issue by tddsg** [syl20bnr/spacemacs#6928]
#### Description
Weird color scheme of `window-numbering` in `spaceline` when running Emacs server (daemon mode).

#### Reproduction guide
- Start Emacs in daemon mode
- Start an Emacs client and notice the color of window-numbering
- Now start Emacs in normal mode.
- Notice the difference of window-numbering color in normal mode with the Emacs in daemon mode.

*Observed behaviour:*
- The Emacs in daemon mode has a very bad color scheme for `window-numbering` section. 
- In the attached picture, it is very hard to the window number in the `daemon` mode
![spacemacs-daemon](https://cloud.githubusercontent.com/assets/20593743/18025880/9475764e-6c69-11e6-94ac-60062b57be3e.png)

- However, the window-numbering scheme of Emacs in `normal` mode is quite good, see the picture below:
![spacemacs-normal](https://cloud.githubusercontent.com/assets/20593743/18025884/bc501afc-6c69-11e6-855b-085b5b7c1294.png)

*Expected behaviour:*
- The window color scheme of both the `daemon` mode and `normal` mode should be consistent.

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.22
- Spacemacs branch: master (rev. 9f9faa4)
- Graphic display: t
- Distribution: spacemacs
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
(auto-completion better-defaults emacs-lisp git markdown org
                 (shell :variables shell-default-height 30 shell-default-position 'bottom)
                 spell-checking syntax-checking version-control semantic latex ocaml haskell tddsg)
```

#### Backtrace
```
BACKTRACE IF RELEVANT
```

**Comment by d12frosted** [comment]
Are you using leuven theme?

**Comment by tddsg** [comment]
Yes, I'm using the `leuven` theme. But this problem also occurs in the `spacemacs-dark` theme:

- `daemon` mode (the window-numbering color scheme is still bad):
![spacemacs-dark-daemon](https://cloud.githubusercontent.com/assets/20593743/18026246/f6d87198-6c72-11e6-93cd-19e56dd7d490.png)

- `normal` mode is OK
![spacemacs-dark-normal](https://cloud.githubusercontent.com/assets/20593743/18026240/e7dca5ce-6c72-11e6-83ca-e6528ea25863.png)

**Comment by TheBB** [comment]
This should have been fixed in develop already.

**Comment by tddsg** [comment]
I switched to the develop branch and saw that the color scheme of window number is good. 
However, the window number up to 2 numbers to display for each window, e.g. 1|1, 1|2... 
I saw it was reported in another issue.

**Comment by TheBB** [comment]
That's the workspace number in addition to the window number.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.

**Comment by tddsg** [comment]
This is wonderful! Thank you for the update!
